WIP

danielkocsis opened the pull request.

Commit:
LPS-87915 portal-search-engine-adapter-api: Add api for suggester

https://issues.liferay.com/browse/LPS-87915

Commit:
LPS-87915 portal-search-elasticsearch6-impl: Implement SuggestExecutor

https://issues.liferay.com/browse/LPS-87915

Commit:
LPS-87915 portal-search-elasticsearch6-impl: Integrate with low lvl suggester api

https://issues.liferay.com/browse/LPS-87915

Commit:
LPS-87915 portal-kernel: Deprecate

https://issues.liferay.com/browse/LPS-87915

Commit:
LPS-87915 portal-search-elasticsearch6-impl: Add test

https://issues.liferay.com/browse/LPS-87915

Commit:
LPS-87915 portal-search-elasticsearch6-impl: Logic that was there previously. Should we be doing this?

https://issues.liferay.com/browse/LPS-87915

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

BryanEngler committed:
LPS-87915 portal-search-elasticsearch6-impl: Update elastic test cases

https://issues.liferay.com/browse/LPS-87915

BryanEngler committed:
LPS-87915 portal-search-elasticsearch6-impl: Return as many suggestions as requested, determined by suggester.setSize() and put exception handling back

https://issues.liferay.com/browse/LPS-87915

BryanEngler committed:
LPS-87915 portal-search-test-util: Update base test case

https://issues.liferay.com/browse/LPS-87915

BryanEngler committed:
LPS-87915 portal-search-engine-adapter-api: Support multiple suggesters and global text in the SuggestSearchRequest

https://issues.liferay.com/browse/LPS-87915

BryanEngler committed:
LPS-87915 portal-kernel: Deprecate AggregateSuggester, no longer needed

https://issues.liferay.com/browse/LPS-87915

BryanEngler committed:
LPS-87915 portal-search-elasticsearch6-impl: Return SuggestionBuilder instead of SuggestBuilder from translators and delete AggregateSuggester

https://issues.liferay.com/browse/LPS-87915

BryanEngler committed:
LPS-87915 portal-search-elasticsearch6-impl: Apply updated APIs

https://issues.liferay.com/browse/LPS-87915

ealonso committed:
LPS-87097 Removes dummy code

Commit:
LPS-90259 Embed personal application portlets in a user page

Commit:
LPS-90259 portal-web - Filter dynamic layout template from list

kyle-miho committed:
LPS-92258 Add functional test for Pages Finder

aline-doleron committed:
LPS-90122 Avoid dom queries on FieldAddedhandler

aline-doleron committed:
LPS-90122 Remove dom queries from LayoutProvider

aline-doleron committed:
LPS-90122 Undo field changes when cancel field changes button is clicked

aline-doleron committed:
LPS-90122 SF

aline-doleron committed:
LPS-90122 Add missing properties

aline-doleron committed:
LPS-90122 Add tests for field resizing

aline-doleron committed:
LPS-90122 Edit test of canceling field changes for a better understanding

aline-doleron committed:
LPS-90122 Remove unused test

aline-doleron committed:
LPS-90122 Update snapshots

aline-doleron committed:
LPS-90122 Remove helptext test since Paragraph field does not have tip attribute

Commit:
LPS-90259 Format user personal application portlets for embedded layout

Commit:
LPS-90259 Hide portlet configuration icons for the control panel portlet

kresimir-coko committed:
LPS-91120 - dynamic-data-mapping - Remove jQuery

Commit:
LPS-86201 LRDOCS-5944 javadoc edit

markocikos committed:
LPS-91120 dynamic-data-mapping - Rename, move before place of use

linolaoj committed:
LPS-90579 setting required properties

Commit:
LPS-90122 Removes unnecessary files

Commit:
LPS-90122 Refactor to return immutable values to avoid unwanted side effects

Commit:
LPS-90122 Optimizations for smoother resizing experience

Commit:
LPS-90122 Updates tests

Commit:
LPS-90122 Updates test snapshots

Commit:
LPS-90259 product-navigation-personal-menu-api - Log exception

ling-alan-huang committed:
LPS-84119 Enable YMLDefinitionOrderCheck

ling-alan-huang committed:
LPS-84119 Add prePrecess and postProcess

ling-alan-huang committed:
LPS-84119 Fix logic to prevent StackOverflow

ling-alan-huang committed:
LPS-84119 Replace string from the accurate position

ling-alan-huang committed:
LPS-84119 Fix

diegonvs committed:
LPS-92734 Use liferay-npm-scripts on document-library-preview-audio

Preston-Crary committed:
LPS-92016 Add support for Prototype AopServices.

Preston-Crary committed:
LPS-92016 Add test coverage.

ling-alan-huang committed:
LPS-84119 Add specific ordering

ling-alan-huang committed:
LPS-84119 Ignore .travis.yml

ling-alan-huang committed:
LPS-84119 Fix regex

ling-alan-huang committed:
LPS-84119 SF

ling-alan-huang committed:
LPS-84119 Don't sort comment

marcoscv committed:
LPS-92799 - Asset publisher new styles

ealonso committed:
LPS-92799 Print should be a button

ealonso committed:
LPS-92799 Convert subscribe into a button and remove icon

ealonso committed:
LPS-92799 Convert conversions links into buttons

ling-alan-huang committed:
LPS-84119 Fix

LPS-84119 Fix

ealonso committed:
LPS-92799 Removes last border from related assets

Commit:
LPS-86201 LRDOCS-5944 @throws tags edit

Commit:
LPS-92838 increase timeout for search tests

diegonvs committed:
LPS-92742 Use liferay-npm-scripts on document-library-web

Commit:
LPS-90259 Show portlet title rather than layout name for embedded personal pages

ling-alan-huang committed:
LPS-84119 Enable tests

ling-alan-huang committed:
LPS-84119 Add testcase

hhuijser committed:
LPS-84119 SF

hhuijser committed:
LPS-84119 Clone the list using 'new ArrayList', now we can order the new list independent of the cloned one and we don't have to convert to String in order to compare

ling-alan-huang committed:
LPS-84119 No need to overwrite the file if it is not formatted

ling-alan-huang committed:
LPS-84119 Remove suppressions

victorg1991 committed:
LPS-87529 Add interface for converting and AssetEntry to a DTO

Co-authored-by: ruben-pulido <ruben.pulido@liferay.com>

victorg1991 committed:
LPS-87529 Create content list API using the AssetEntryToDTOConverter to avoid duplicating code

Co-authored-by: ruben-pulido <ruben.pulido@liferay.com>

victorg1991 committed:
LPS-87529 Extract toDTO logic to Util and use if from converters.
Write converters for StructuredContent, Folder and BlogPosting for now.

Co-authored-by: ruben-pulido <ruben.pulido@liferay.com>

diegonvs committed:
LPS-92941 Use liferay-npm-scripts in map-openstreetmap

diegonvs committed:
LPS-92732 Use liferay-npm-scripts on document-library-preview-image

diegonvs committed:
LPS-92733 Use liferay-npm-scripts in document-library-preview-video

Commit:
LPS-90259 Extract isEmbeddedPersonalApplicationLayout() to BasePortletConfigurationIcon

Commit:
LPS-92351 Add DDMStructureLayout search contributor files

Commit:
LPS-92351 Add search context

Commit:
LPS-92531 Add custom method

Commit:
LPS-92351 Service builder gen

Commit:
LPS-92531 Add custom method

diegonvs committed:
LPS-92959 Use liferay-npm-scripts in map-google-maps

ruben-pulido committed:
LPS-87529 Track all the services implementing the interface, this way supported assets can be contributed in a decoupled way

Co-authored-by: victorg1991 <victor.galan@liferay.com>

diegonvs committed:
LPS-92968 Use liferay-npm-scripts on users-admin-web

pyoo47 committed:
LRCI-36 Fully implement build properties caching

pyoo47 committed:
LRCI-36 Include stopWatchRecords that have no duration

pyoo47 committed:
LRCI-36 Fix ConcurrentModification exception in redact()

diegonvs committed:
LPS-92968 Use liferay-npm-scripts inside site-admin-web

petershin committed:
LPS-92936 Use the default scriptFile when the package-lock.json file exists

jbalsas committed:
LPS-93009 Adds liferay-npm-scripts to the roow workspace and removes it as a dev dependency from the other workspaces

michaelhashimoto committed:
LRCI-108 Add a new java based job for portal-testsuite-upstream-controller

brianchandotcom committed:
LPS-87529 Simplify

ivicac committed:
COMMERCE-1217 - portal-vulcan-impl - Move logic from JSONMessageReader/Writer to base classes

ivicac committed:
COMMERCE-1217 - portal-vulcan-impl - Add XMLMessageBodyReader/Writer

ivicac committed:
COMMERCE-1217 - portal-vulcan-impl - Add XmlMapperContextResolver

ivicac committed:
COMMERCE-1217 - portal-vulcan-impl - Register XML related providers

ivicac committed:
COMMERCE-1217 - portal-vulcan-impl - Update dependencies

ivicac committed:
COMMERCE-1217 - portal-vulcan-test - Add tests

brianchandotcom committed:
LPS-87529 Don't generate

brianchandotcom committed:
LPS-87529 make it a util

brianchandotcom committed:
LPS-87529 regen

brianchandotcom committed:
LPS-87529 Use new util

ling-alan-huang committed:
LPS-84119 Auto SF

p2kmgcl committed:
LPS-92959 Remove devDependencies

p2kmgcl committed:
LPS-92968 Remove devDependencies

ruben-pulido committed:
LPS-87529 Pass base resource class instead of resource class when calling
JaxRsLinkUtil.getJaxRsLink() since the base resource class is the one
with @Path annotations

jbalsas committed:
LPS-93009 Updates lock file

adolfopa committed:
LPS-90113 Use new "move" Lexicon icon in D&M

adolfopa committed:
LPS-90113 Use new "move" Lexicon icon in Fragments and Journal

ruben-pulido committed:
LPS-87529 Remove unused fields

ruben-pulido committed:
LPS-87529 Register StructuredContentDTOConverter component as a service also under
 the type StructuredContentDTOConverter

pyoo47 committed:
LRCI-36 Simplify

Commit:
LPS-91855 Fix wrong import package

pavel-savinov committed:
LPS-93029 Use layout page template entry associated with published layout

Commit:
LPS-90259 Layout of type control panel is not an embedded personal application layout

drewbrokke committed:
LPS-92968 users-admin-web - package.json - adds SF exception to allow console.error logging

Commit:
LPS-91855 Simplify/Generalize InfoItemRenderer

The options for how to render, such as whether a template can be specify, do not always make sense. By removing it implementors now have two options:
1) To implement several renderers, one per template/variation
2) For a dynamic list of options, create a "Wrapper" object that includes the entity to render along with its variations

Commit:
LPS-91855 Adapt AssetEntry renderer to the simplified/generalized InfoItemRenderer

brianchandotcom committed:
LPS-92351 new pattern, see 4576487848ce35ac48b8dc8af060c15f52640ef7

brianchandotcom committed:
LPS-92351 Sort

drewbrokke committed:
LPS-92968 users-admin-web - SF - fixes csf failures

brianchandotcom committed:
LPS-92351 We never call it this in our other *ServiceImpl, type: git grep "search(" **ServiceImpl.java

brianchandotcom committed:
LPS-92351 regen

joshchong committed:
LRQA-47758 Remove workaround

brianchandotcom committed:
LRCI-108 prep next

brianchandotcom committed:
COMMERCE-1217 Sort

codyhoag committed:
LPS-86201 LRDOCS-5944 Edit Javadoc

pavel-savinov committed:
LPS-92957 Layout can be null, avoid NPE

codyhoag committed:
LPS-86201 LRDOCS-5944 Format Javadoc

brianchandotcom committed:
COMMERCE-1217 SF

pavel-savinov committed:
LPS-92996 Freemarker allows only alphanumeric and underscore characters in the variable names

ealonso committed:
LPS-92913 Makes ddm structure form fluid

codyhoag committed:
LPS-90081 LRDOCS-6452 Edit tlddoc

Commit:
LPS-91855 Make more explicit that renderers must have a unique key and can be retrieved by it

Still provide a default implementation which uses the class name as the key

michaelhashimoto committed:
LRCI-108 Add getJobURL to BuildData objects

Commit:
LPS-91855 Make it easier to obtain renders for a given type of item by requiring the className and not a Class instance

michaelhashimoto committed:
LRCI-108 Add a method to keep the logs of a given build

michaelhashimoto committed:
LRCI-108 Add a new BuildData class for TestSuite Upstream builds

michaelhashimoto committed:
LRCI-108 Only keep builds that have a new SHA

jbalsas committed:
LPS-93009 Adds top level format scripts for future automation

brianchandotcom committed:
LPS-92351 fix.. my bad

brianchandotcom committed:
LPS-866638 it is trying to match 'sLAId' with 'slaId' and so fails - Peter, I need a better fix for this. In modules/private/apps/portal-workflow/portal-workflow-metrics-rest-impl/rest-openapi.yaml, the schema is SLA, the id is slaId, but we see a lot of sLAId. Please fix and revert this.

brianchandotcom committed:
LPS-92968 SOrt constants first, if this was hard coded, can we just do cmd: cmd, if so, then why do we need '' around cmd?

jpince committed:
LRQA-47868 Temporarily remove upgrade tests from relevant

brianchandotcom committed:
LPS-92968 Listen to CSF

mtambara committed:
LPS-92982 Make these dependencies compileOnly

brianchandotcom committed:
LPS-91855 rename

Commit:
LPS-92837 Sync connector admin settings form h4 should be h3

joshchong committed:
LRQA-47867 Update variable due to LPS-92248

brianchandotcom committed:
LPS-88218 Allow function

brianchandotcom committed:
Baseline

brianchandotcom committed:
Revert "LPS-88218 Allow function"

This reverts commit 3c694aaf4c905172b1ae59d0a8515903fb311916.

ealonso committed:
LPS-92913 Adapt image to new styles

ealonso committed:
LPS-92913 Removes custom styles for image content

ealonso committed:
LPS-92913 Not needed

ealonso committed:
LPS-92913 Uses css classes instead of custom styles

shuyangzhou committed:
LPS-92982 Make all dependencies compileOnly

shuyangzhou committed:
LPS-92982 Fix transitive dependency users.

pavel-savinov committed:
LPS-92126 Add check method to throw exception directly without boolean return

pavel-savinov committed:
LPS-92126 Check layout page template entry permissions separately from layout prototypes

ealonso committed:
LPS-82343 BuildLang

jkappler committed:
LPS-92248 Fix language key message

jkappler committed:
LPS-92248 Consistent naming

jkappler committed:
LPS-92248 Build lang - layout type controller asset display

ealonso committed:
LPS-91836 Adds support to delete thumbnails into fragmets

ealonso committed:
LPS-91836 Adds support to delete thumbnails into layout page templates

jkappler committed:
LPS-92248 Build lang - portal impl

ealonso committed:
LPS-91836 Language keys

ealonso committed:
LPS-91836 buildLang

Commit:
LPS-92446 Missing export

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-92703 Change logic to keep the right behavior for "hidden"

Commit:
LPS-92703 Provide explanatory help message

javiergamarra committed:
LPS-92096 update question also in update

javiergamarra committed:
LPS-93015 add creator property

javiergamarra committed:
LPS-93015 generate

javiergamarra committed:
LPS-93015 expose creator information in KBFolder

javiergamarra committed:
LPS-93005 add encoding format to articles and messages

javiergamarra committed:
LPS-93005 generate

javiergamarra committed:
LPS-93005 adapt to new property

javiergamarra committed:
LPS-93006 fix validation error, path params need required in OpenAPI

javiergamarra committed:
LPS-93006 add right filter to show KB articles from root

javiergamarra committed:
LPS-93006 use right key

javiergamarra committed:
LPS-89821 use right locales

javiergamarra committed:
LPS-92725 add default message also in PUT

javiergamarra committed:
LPS-83885 expose more fields

javiergamarra committed:
LPS-83878 add filtering in organization

javiergamarra committed:
LPS-83878 generate

javiergamarra committed:
LPS-83878 add filtering/sorting in organization API resource

shuyangzhou committed:
LPS-91855 You can not have abstract Component, caused by 05e8915a82493429715f7e97efe34690bb8abf95

petershin committed:
LPS-93083 Automatically disable the jar task for xxx-test projects

petershin committed:
LPS-93084 Fix parameter

petershin committed:
LPS-93085 Set Gradle HTTP timeouts to 120s for Jenkins

petershin committed:
LPS-93083 Auto SF

javiergamarra committed:
LPS-81355 add ratings to threads, messages, documents, articles, kbarticles and blogs

javiergamarra committed:
LPS-81355 generate

javiergamarra committed:
LPS-81355 add common resource class to use in webm, collab and dm

javiergamarra committed:
LPS-81355 add common resources and util converter classes to each suite

javiergamarra committed:
LPS-81355 expose post and get page to all the resources with ratings

javiergamarra committed:
LPS-91225 emails of a user/org does not have pagination

javiergamarra committed:
LPS-91225 generate

javiergamarra committed:
LPS-91225 adapt

javiergamarra committed:
LPS-91225 implementation of email tests

javiergamarra committed:
LPS-91225 fix small bug in equals check

brianchandotcom committed:
LPS-92725 This is fine, since we don't need to reference the original one after the update

brianchandotcom committed:
LPS-81355 Sort

brianchandotcom committed:
LPS-81355 Good enough

brianchandotcom committed:
LPS-81355 SF

brianchandotcom committed:
LPS-81355 auto SF

brianchandotcom committed:
LPS-91225 Sort

brianchandotcom committed:
LPS-91836 SF

brianchandotcom committed:
LPS-91836 SF

brianchandotcom committed:
Regen

brianchandotcom committed:
LPS-93083 prep next

brianchandotcom committed:
LPS-90259 Sort

brianchandotcom committed:
LPS-90259 Sort

brianchandotcom committed:
LPS-90259 SF

brianchandotcom committed:
LPS-90259 Buy space

brianchandotcom committed:
LPS-90259 auto SF

brianchandotcom committed:
LPS-92703 Wordsmith

brianchandotcom committed:
LRCI-108 prep next

brianchandotcom committed:
LPS-87529 update because of changes from LPS-92392

brianchandotcom committed:
LPS-87529 only prefix with context* for the auto generated ones in Base*ResourceImpl

javiergamarra committed:
LPS-87529 move API to fix tests

javiergamarra committed:
LPS-87529 generate

javiergamarra committed:
LPS-87529 move code to ratings

brianchandotcom committed:
LPS-87529 I think it should go in headless-web-expereince-api. I'm fine with the other headless* depending on headless-web-experience-api

brianchandotcom committed:
LPS-87529 Make it explicit

brianchandotcom committed:
LPS-81355 only do tests when Page returns the same type

brianchandotcom committed:
LPS-81355 regen

brianchandotcom committed:
LPS-81355 Remove Rating.class as a className for the other uses. It's very confusing since it can be wrong.

brianchandotcom committed:
Revert "LPS-81355 only do tests when Page returns the same type"

This reverts commit 11a5bba3b7be0db3b604590a624579aacf5f3e63.

brianchandotcom committed:
Revert "LPS-81355 regen"

This reverts commit feda8b9d971db52e52c4fb50565889bc19d047a4.

brianchandotcom committed:
LPS-87529 fix rebase error

brianchandotcom committed:
LPS-84119 regen

brianchandotcom committed:
LPS-87529 Fix for collab

brianchandotcom committed:
LPS-87529 regen

brianchandotcom committed:
LPS-87529 fix for doclib

brianchandotcom committed:
LPS-87529 regen

brianchandotcom committed:
LPS-84119 regen

brianchandotcom committed:
LPS-84119 prep next

brianchandotcom committed:
LPS-84119 prep next

shuyangzhou committed:
LPS-92982 These two still need to be compile for now.

brianchandotcom committed:
LPS-84119 ran auto SF

brianchandotcom committed:
LPS-84119 regen

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/analytics/.gitrepo'.

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 1921b14a30564fe22db4b69a1ed327e9611e00f7

jkappler committed:
LPS-84119 Use variable for defining css style in soy

Commit:
Handling related portlet preferences when updating a layout

Commit:
Autogenerated

Commit:
WIP
